.github/workflows/ci.yml

```diff
@@ -3,60 +3,31 @@ on: [push, pull_request]
 jobs:
   test-elixir:
     runs-on: ubuntu-latest
-    container:
-      image: hexpm/elixir:${{ matrix.elixir }}-erlang-${{ matrix.otp }}-alpine-3.12.0
-      env:
-        ELIXIR_ASSERT_TIMEOUT: 2000
-
+    env:
+      GIT_URL_INSTEAD_OF: "https://github.com/=git@github.com:"
```

Comment:
I was reading the docs and it looks like this option might be obsolete now:

https://github.com/earthly/earthly/blob/main/docs/earthly-command/earthly-command.md#--git-url-instead-of-git-instead-of-obsolete